src/Gallery/Gallery.js

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import './Gallery.css';
+
+import { NavLink, Redirect } from 'react-router-dom';
+import PropTypes from 'prop-types';
+
+const Gallery = (props) => {
+  if (props.art) {
+    return (
+      <div style={{ backgroundColor: props.currentColor, textAlign: "center" }}>
+        <NavLink to='/' className='home-btn'>home</NavLink>
+        <button className='home-btn' onClick={() => window.history.back()}>back</button>
+        <section className='gallery'>
+          {props.art.records.map(piece => {
+        return (
+          
+            <article className='art-icle' key={piece.objectid}>
+            <img 
+              className='art-img' 
+              src={piece.primaryimageurl ? piece.primaryimageurl : require('./lostimg.jpg')} 
```

Comment:
Really hoping to figure out better error handling for broken image `src`s, even though I do like the filler image I slapped together.